### DISCUSSION TITLE
Could not compile basic sample

### DISCUSSION THREAD

**Discussion by alexandrefelipemuller** [AppIntro/AppIntro#987] [General]
Hi, I'm trying to put AppIntro on my app, I add it to build.gradle and could sync
But when I use the basic sample provided it show an error like:
Unresolved reference: createInstance

My code
```
import android.os.Bundle
import androidx.fragment.app.Fragment
import com.github.appintro.AppIntro
import com.github.appintro.AppIntroFragment

class Tutorial : AppIntro() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        // Make sure you don't call setContentView!

        // Call addSlide passing your Fragments.
        // You can use AppIntroFragment to use a pre-built fragment
        addSlide(
            AppIntroFragment.createInstance(
            title = "Welcome...",
            description = "This is the first slide of the example"
        ))
        addSlide(AppIntroFragment.createInstance(
            title = "...Let's get started!",
            description = "This is the last slide, I won't annoy you more :)"
        ))
    }
```
Another error that get my attention is that it show: Overriding method should call super.onCreate

But it is already.. so I'm confused what it could be

**Comment by cortinico** [comment]
Your code looks fine. Coudl you share your project publicly or create a reproducer?

**Reply by alexandrefelipemuller** [reply]
Thank you for prompt reply. 
Of course I can share, my app is: https://github.com/alexandrefelipemuller/plantatreeapp
the error is in Tutorial.kt
https://github.com/alexandrefelipemuller/plantatreeapp/blob/main/app/src/main/java/org/arboristasurbanos/treeplant/ui/tutorial/Tutorial.kt
It shows like cannot find createInstance.. and the onCreate show some strange errors.

**Comment by paolorotolo** [comment]
You're right, I'm sorry for the inconvenience.
It seems https://github.com/AppIntro/AppIntro/commit/561a7af0c64c3bd10735e90f385e35a99fa610eb updated README with unreleased changes.

You have 2 ways to fix this:
 * Refer to a previous version of README (https://github.com/AppIntro/AppIntro/blob/6.1.0/README.md) and use `AppIntroFragment.newInstance` instead of `AppIntroFragment.createInstance`;
 * Use latest AppIntro unreleased snapshot from here: https://jitpack.io/#AppIntro/appintro/-SNAPSHOT

@cortinico it's time to finally merge https://github.com/AppIntro/AppIntro/pull/978 😄

**Reply by alexandrefelipemuller** [reply]
Thank you guys! you are awesome

**Reply by cortinico** [reply]
> @cortinico it's time to finally merge #978 😄

Agree. I'll try to do it in the next days 👌
